README.md

```diff
@@ -1,42 +1,139 @@
-# my-package-name
+# openapi-client
+https://www.stitchdata.com/docs/developers/stitch-connect/api
 
-[![](https://img.shields.io/pypi/v/my-package-name.svg)](https://pypi.org/pypi/my-package-name/) [![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
+This Python package is automatically generated by the [OpenAPI Generator](https://openapi-generator.tech) project:
 
-<!-- Short description of the package -->
+- API version: 0.1
```

Comment:
Is there a way to customize this README by passing configuration to the generator?  Or even have it generate a different file, like GUIDE.md so that we could keep control over the main README?

Alternatively, we could nest the entire auto-generated folder so that the top level repo only included a README and subfolder..

Reply:
as far as i can tell we can't customize the README without creating a custom generator, what's your opinion on manually renaming vs nesting the entire generated folder

Reply:
Probably nesting is less maintenance?